iStory/Scenes/Authentication/SignUp/AuthenticationIstorySignUpViewController.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  AuthenticationIstorySignUpViewController.swift
+//  iStory
+//
+//  Created by Nikola Stojanovic on 2.4.22..
+//
+
+import UIKit
+
+final class AuthenticationIstorySignUpViewController: UIViewController {
+
+    override func loadView() {
+        super.loadView()
+        let viewModel = AuthenticationIstoryViewModel(title: String(localized: "splash.auth.signup.title"),
```

Comment:
I think istory is not necessary, AuthenticationViewModel is enough if you ask me. What do you think?